src/controllers/profile.js

```diff
@@ -0,0 +1,110 @@
+import { config } from 'dotenv';
+import cloudinary from 'cloudinary';
+import { User } from '../sequelize/models';
+
+config();
+cloudinary.config({
+  cloud_name: process.env.cloudinaryName,
+  api_key: process.env.cloudinaryApiKey,
+  api_secret: process.env.cloudinarySecretKey,
+});
```

Comment:
I think it would be good that we all adhere to the convention on naming environment variable by using `_` instead of camel case like in this case. 
<img width="356" alt="Screen Shot 2019-08-13 at 14 45 02" src="https://user-images.githubusercontent.com/50101879/62943303-71b94c00-bdda-11e9-8115-8c32536a0f98.png">
So instead of `cloudinaryName`, u can use `CLOUDINARY_NAME`

Reply:
> I think it would be good that we all adhere to the convention on naming environment variable by using `_` instead of camel case like in this case.
> <img alt="Screen Shot 2019-08-13 at 14 45 02" width="356" src="https://user-images.githubusercontent.com/50101879/62943303-71b94c00-bdda-11e9-8115-8c32536a0f98.png">
> So instead of `cloudinaryName`, u can use `CLOUDINARY_NAME`

Thank you. On it now